src/components/MerchantsPage/gam/MegaGamListItem/MegaGamListItem.tsx

```diff
@@ -35,9 +39,15 @@ const MegaGamListItem = ({ campaign }: Props) => {
           <LearnMoreText>
             Learn more about our participating merchants:
           </LearnMoreText>
-          {/* TODO: Add seller/distributor pairs. */}
           <SellerDistributorPairs>
-            _SELLER_DISTRIBUTOR_PAIRS_
+            {campaign.seller_distributor_pairs.map(
+              (sellerDistributorPair: SellerDistributorPairType) => (
+                <SellerDistributorPair
+                  key={`${sellerDistributorPair.distributor_id};${sellerDistributorPair.seller_id}`}
```

Comment:
This notation is kind of strange to me

Reply:
Is this the key? I think the point is to combine the distributor_id and seller_id as the unique key with a semi colon as a separator instead of concatenating

Reply:
yeah this is the key. it was operating under the previous assumption i had about seller_id (int). it was to distinguish between a row w/ distributor id 11 and seller id 1 and a row w/ distributor id 1 and seller id 11. it's moot now tho